GoogleForCreators/web-stories-wp
# Editor: Fixed empty story state when there is an attachment

## Context


An empty Story text and icon should disappear when an attachment is added to a Story.

## Summary


Currently, even if the Page Attachment is added, the Empty State helper is still shown — but it shouldn’t, since the Page is already changed really.

You can add the Page Attachment locally by:
1. Create a new story.
2. Click on the “Background” list item in the right bottom corner of the screen (The one under “Layers”).
3. Now the Page gets selected and you should be able to see the “Page Attachment” section in the right-side panel.




## User-facing changes


Before: 
<img width="1012" alt="Screenshot 2022-03-09 at 15 32 19" src="https://user-images.githubusercontent.com/2496186/157483079-6d83a03b-96f7-4d7b-8c82-acdacffff943.png">
After:

![image](https://user-images.githubusercontent.com/2496186/157483213-51282d26-94bb-4cb8-a5ea-2da9a4942a7a.png)


## Testing Instructions


This PR can be tested by following these steps:

1. Create 1 Story.
2. Add a link in the Attachments.
3. Empty story text and icon should disappear.






## Checklist




---



Fixes #10700
